Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/JointLimitRotationManipulators.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/Viewport/ViewportColors.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+#include <EMotionFX/CommandSystem/Source/JointLimitCommands.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/Plugins/Ragdoll/JointLimitRotationManipulators.h>
+#include <Editor/Plugins/Ragdoll/RagdollJointLimitWidget.h>
+
+namespace EMotionFX
+{
+    JointLimitRotationManipulators::JointLimitRotationManipulators(JointLimitFrame jointLimitFrame)
+        : m_rotationManipulators(AZ::Transform::Identity())
```

Comment:
nit: We could use a member initializer instead and remove this from the constructor initializer list

Reply:
Done.